frontend/src/crosswordView/crosswordView.scss

```diff
@@ -0,0 +1,121 @@
+.crossword-title {
+    font-size: 50px;
+}
+
+.give-up {
+    margin: 10px 10px 10px 0;
+}
+
+.difficulty-button {
+    height: 100px;
+    width: 200px;
+    font-size: 40px;
+
+}
+
+// Clue Styles
+.clues-box {
+    background: rgba(105, 64, 238, 0.25);
+    border-radius: 10px;
+    padding: 0 10px 10px;
+    margin-bottom: 30px;
+    max-height: 300px;
+    overflow-y: scroll;
+}
+
+.clue-header {
+    margin: 10px 0;
+}
+
+.clue {
+    margin: 5px 0 5px 15px;
+}
+
+.clue-found {
+    text-decoration: line-through;
+    color: #a0a0a0;
+}
+
+// Definition Box
+.definitions {
+    background: rgba(105, 64, 238, 0.25);
+    border-radius: 10px;
+    padding-bottom: 20px;
+    max-height: 300px;
+    overflow-y: scroll;
+}
+
+.definition-title {
+
+}
+
+// Crossword Styles
+.crossword-grid {
+    display: flex;
+    flex-direction: column;
+}
+
+.crossword-row {
+    display: flex;
+    justify-content: center;
+}
+
+$cell-width: 50px;
+$cell-height: 50px;
```

Comment:
This is a matter of personal style, probably, but I think it's a good idea to pull up SCSS variable assignments to the top of the file so that it's easy for our future selves to find them.